### PR TITLE
Added instructions for installing stable RPM packages.

### PR DESCRIPTION
**Pull request by BClark09**
Signed-off-by: Ben Clark <ben@benjyc.uk>

### CODE REVIEW

File: installation/linux.md [hunk 1]
```diff
@@ -161,21 +159,53 @@ Yum or Dnf Based Systems
 
 {% include collapsible/body.html %}
 
-**Note:** The RPM files within the cannot currently be GPG signed or checked, only continue if you're happy installing without GPG verification.
+**Note:** The RPM files cannot currently be GPG signed or checked, only continue if you're happy installing without GPG verification.
 
-The RPM repo information should be defined inside a file, make a new file at `/etc/yum.repos.d/openhab.repo` with the following contents:
+The RPM repo information should be defined inside a file, make a new file at `/etc/yum.repos.d/openhab.repo` with the following contents depending on your choice of release.
+You may add all three to the same file, but make sure the desired repo is is set to `enabled=0`:
```
Comment (ThomDietrich) [inline installation/linux.md]: 1?

File: installation/linux.md [hunk 2]
```diff
@@ -161,21 +159,53 @@ Yum or Dnf Based Systems
...
-repo_gpgcheck=0
-gpgkey=https://openhab.jfrog.io/openhab/api/gpg/key/public 
-```
+* **Official Release**
+
+  The stable builds contain the latest official release with tested features.
+
+  ```text
+  [openHAB-Release]
+  name=openHAB 2.x.x Release
```
Comment (ThomDietrich) [inline installation/linux.md]: Is release contrary to testing? How about 

* Stable Release
* Testing Release
* Snapshot/Unstable Release

**Comment by BClark09** [comment]
Right you are! :) Hope that's better.

**Comment by ThomDietrich** [comment]
Four eyes principle ftw ;D

**Pull request merged**